dashboard/app/assets/javascripts/header.js.erb

```diff
@@ -413,3 +419,35 @@ dashboard.header.updateTimestamp = function () {
     $('.project_updated_at').text("Not saved"); // TODO i18n
   }
 };
+
+function getSummarizedProgressForAnonymousUser () {
+  var summarizedProgress = {};
+  var levelProgress = {};
+
+  summarizedProgress.lines = $.cookie('lines');
+
+  var allLevelsProgress = JSON.parse(decodeURIComponent($.cookie('progress')));
+  for(var level in allLevelsProgress) {
+    var individualLevelProgressDescription = '';
```

Comment:
These do appear to get minified on production - they end up in application-<digest>.js. Names on the dashboard namespace will not be minified (i.e. you can search for dashboard.buildHeader), but local function names will be (i.e. you won't find a showHeaderPopup).